src/dos/cdrom.h

```diff
@@ -143,6 +143,7 @@ class CDROM_Interface_Image : public CDROM_Interface
 		uint32_t adjustOverRead(const uint32_t offset,
 		                        const uint32_t requested_bytes);
 		int length_redbook_bytes = -1;
+		uint32_t audio_pos = (std::numeric_limits<uint32_t>::max)(); // last position when playing audio
```

Comment:
There's no need for parenthesis around `::max` any more :)

Reply:
Excellent! I thought it was just the simpler `min(a, b)`, `max(a, b)` functions; great to see us not pinned down by MSVC-custom stuff anymore.

Reply:
Fixed in a401248b

Reply:
Fixed in 9963a43.